input/pagecontent/index.md

```diff
@@ -32,7 +32,7 @@ This guide is divided into several pages which are listed at the top of each pag
 - [FHIR Artefacts](artifacts.html): These pages provide detailed descriptions and formal definitions for all the FHIR artefacts defined in this guide.
   - [Profiles and Extensions](profiles-and-extensions.html): This set of pages describes the profiles and extensions that are defined in this guide to exchange quality data. Each profile page includes a narrative description and guidance, formal definition and a "Quick Start" guide which summarises the supported search transactions for each profile. Although the guidance typically focuses on the profiled elements, it may also may focus on un-profiled elements to aid with implementation.
   - [Terminology](terminology.html): This set of pages lists the value sets and code systems defined in this guide.
-- [Examples](examples.html): This is a placeholder for when content becomes available.
-- [Downloads](downloads.html): This is a placeholder for when content becomes available.
+- [Examples](examples.html): This page lists all the examples used in this guide.
+- [Downloads](downloads.html): This page provides links to downloadable artefacts that developers can use to help them implement this guide.
```

Comment:
Need to either do the full change or revert this half improvement of the downloads page line and leave the fix to another PR that updates this fully in line with changes to page organisation from HL7 AU licensing.

Reply:
Reverted the change related to Downloads page